source/source.go

```diff
@@ -0,0 +1,146 @@
+// Copyright © 2022 Meroxa, Inc & Yalantis.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package source
+
+import (
+	"context"
+	"fmt"
+
+	sdk "github.com/conduitio/conduit-connector-sdk"
+	"github.com/jmoiron/sqlx"
+
+	_ "github.com/ibmdb/go_ibm_db" //nolint:revive,nolintlint
+
+	"github.com/conduitio-labs/conduit-connector-db2/config"
+	"github.com/conduitio-labs/conduit-connector-db2/source/iterator"
+)
+
+// Source connector.
+type Source struct {
+	sdk.UnimplementedSource
+
+	config   Config
+	iterator Iterator
+}
+
+// New initialises a new source.
+func New() sdk.Source {
+	return &Source{}
+}
+
+// Parameters returns a map of named sdk.Parameters that describe how to configure the Source.
+func (s *Source) Parameters() map[string]sdk.Parameter {
+	return map[string]sdk.Parameter{
+		config.KeyConnection: {
+			Description: "Connection string to DB2",
+			Required:    true,
+			Default:     "",
+		},
+		config.KeyTable: {
+			Description: "A name of the table that the connector should write to.",
+			Required:    true,
+			Default:     "",
+		},
+		KeyPrimaryKeys: {
+			Description: "A names of columns that connector will use for create record key",
```

Comment:
A names of columns > Names of columns.

Or: Columns that will form a record key.

Reply:
fixed